components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesEndpoint.java

```diff
@@ -76,6 +74,11 @@ public void startRoute(String id) throws Exception {
         camelContext.getRouteController().startRoute(id);
     }
 
+    public void resetRoute(String id) throws Exception {
+        ManagedRouteMBean resetRoute = camelContext.getManagedRoute(id, ManagedRouteMBean.class);
+        resetRoute.reset();
```

Comment:
My thoughts on this are, with start/stop/resume/ we are probably just logging the info for what camel does with route id's and not some status code.  I think only having a message show up with 204 should be fine for users to understand and provide a proper and correct route id.  
Guys, let me know if any further changes need to be done.